### PR TITLE
Add an unrolled loop and swap GR/LW A/B order

### PR DESCRIPTION
**Pull request by hcman2**
Add UnrollLoopSwapGlobalReadOrder feature.
TODO:
Add a test yaml.
Validate the CI cases.

### CODE REVIEW

**Comment by hcman2** [comment]
test yaml added.

**Comment by vin-huang** [comment]
The definition of UseScaleAlphaVec has been changed as below, please kindly modified your YAML file accordingly.

UseScaleAlphaVec=False -> UseScaleAlphaVec=0

UseScaleAlphaVec=True -> UseScaleAlphaVec=1

Redefined UseScaleAlphaVec
0 - Alpha Vector Scaling is disabled
1 - Calculated Alpha Vector Scaling along M-dim
2 - Calculated Alpha Vector Scaling along N-dim
3 - Calculated Alpha Vector Scaling along either M-dim or N-dim.

**Review by aazz44ss — APPROVED** [review]
LGTM

**Review by KKyang — APPROVED** [review]

File: tensilelite/Tensile/KernelWriter.py [hunk 1]
```diff
@@ -1587,13 +1587,17 @@ def noLoadLoopBody( self, kernel, tensorParametersA, tensorParametersB, pack, is
...
       if u == 0:
         if not isLastLoop:
-          self.codes.localWriteA = self.localWriteDo(kernel, tensorParametersA)  # local write in loopcnt N targets data for loopcnt N+1
-          self.codes.localWriteB = self.localWriteDo(kernel, tensorParametersB)
+          if dsWriteBA==True:
+            self.codes.localWriteA = self.localWriteDo(kernel, tensorParametersB, swapAB=1)
+            self.codes.localWriteB = self.localWriteDo(kernel, tensorParametersA, swapAB=1)
+          else:
+            self.codes.localWriteA = self.localWriteDo(kernel, tensorParametersA)  # local write in loopcnt N targets data for loopcnt N+1
+            self.codes.localWriteB = self.localWriteDo(kernel, tensorParametersB)
```
Comment (hcman2) [inline tensilelite/Tensile/KernelWriter.py:1595]: In the current scheduling, we always schedule lwa first then lwb second. 
Put B in lwa code can easily change the order.

**Pull request merged**